### PR TITLE
add print_tree_ascii function to TreeTools

### PR DESCRIPTION
**Pull request by anna-parker**
screenshot_draw_ascii
Tees can now be printed in the terminal as in Bio.Phylo.draw_ascii using the print_tree_ascii command directly in julia without having to use a python wrapper (PyCall). I noticed PyCall has some strange behavior changes when moving from VScode to jupyter and vice versa so thought it might be good to add this.

### CODE REVIEW

File: src/misc.jl [hunk 1]
```diff
@@ -81,12 +81,107 @@ end
...
+        println(line)
+    end
+end
+
+"""
+    distance_from_root(node::TreeNode)
+
+    in added branch lengths
+"""
+function distance_from_root(node::TreeNode)
```
Comment (PierreBarrat) [inline src/misc.jl:174]: There is already a `divtime` function that computes the branch length distance between two nodes (now that I think about it, it should be renamed to `distance`). One would call it like `divtime(node, root)`. 
If you think it's practical to have a separate `distance_to_root` function, you could move it to `methods.jl`, along with other "distance" functions.

**Comment by PierreBarrat** [comment]
Nice!! Some comments: 

1. Does not work for trees that have `missing` branch length. 
Capture du 2022-04-11 17-22-13

  Maybe we could assign a default branch length in this case? 

2. Since this display of trees is much nicer than the current one, we should change the `show(t::Tree)` function to call `print_tree_ascii`

**Comment by anna-parker** [comment]
Hi! I hope this sorts out your comments, I have just opted for the easy approach when any branch length is unknown the tree will be constructed using unit branch lengths and will print that it is doing so, as shown below:
ascii_fix

**Comment by PierreBarrat** [comment]
Looking good  :smiley:

**Pull request merged**